dao/src/main/java/com/epam/esm/dao/builder/BuilderSql.java

```diff
@@ -0,0 +1,88 @@
+package com.epam.esm.dao.builder;
+
+import com.epam.esm.model.parameters.CertificateParameter;
+import com.epam.esm.model.parameters.CertificateSortType;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BuilderSql {
+    public static final String PERCENT_SIGN = "%";
+    public static final String ORDER = "ORDER BY";
+    public static final String ASC = "ASC";
+    public static final String DESC = "DESC";
+    public static final String LIKE_OPERAND = "like";
+    private static final String PARAMETER_SORT = "sort";
+    public static final String EQUALS = "=";
+    public static final String COMMA = ",";
+    public static final String SPACE = " ";
+    public static final String EMPTY = "";
+    private static final char MINUS = '-';
+    private static final String AND = "AND";
+    private static final String QUOTES="\"";
+    private static final String WHERE="where";
+
+    public static String buildFindAndSortCertificateByParameter(Map<String, String> groupParameters) {
+        StringBuilder result = new StringBuilder();
+        Map<String, String> groupParams = groupParameters.entrySet().stream().
+                filter(s -> !s.getKey().toUpperCase().equals(PARAMETER_SORT.toUpperCase())).
```

Comment:
you could use `equalsIgnoreCase` instead of twice usage of `toUpperCase`

Reply:
done